### DISCUSSION TITLE
Welcome to frappe-ui Discussions!

### DISCUSSION THREAD

**Discussion by netchampfaris** [frappe/frappe-ui#35] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by vignesh235** [comment]
how use frappe ui in site

**Comment by chechani** [comment]
Hi , Request to let us know Whether Frappe UI can be used as a standalone vuejs-based app , means not part of the custom frappe app.

**Comment by samarsingh6965** [comment]
Can i use frappe UI using a cdn link?
if Yes! How?

**Comment by priteshfps** [comment]
I have created custom app with vuejs front-end I want to check every minute frappe session is active or not which Api can I use and at the time of checking does not want refresh session
I have tried frappe.auth.get_logged_user but it is refreshing session
